core/src/test/scala/org/apache/spark/memory/UnifiedMemoryManagerSuite.scala

```diff
@@ -40,10 +42,12 @@ class UnifiedMemoryManagerSuite extends MemoryManagerSuite with PrivateMethodTes
   override protected def createMemoryManager(
       maxOnHeapExecutionMemory: Long,
       maxOffHeapExecutionMemory: Long): UnifiedMemoryManager = {
+    val offHeapSize = sc.get(MEMORY_OFFHEAP_SIZE)
```

Comment:
I think we always get default value here.

Reply:
You are right.